hypothesis-python/tests/nocover/test_build_signature.py

```diff
@@ -66,9 +66,27 @@ def __init__(self, **kwargs):
         assert set(kwargs) == {"testA", "testX", "testY"}
         assert isinstance(kwargs["testA"], int)
         assert isinstance(kwargs["testX"], float)
-        assert isinstance(kwargs["testY"], str)
+        assert isinstance(kwargs["testY"], list)
+        assert all(isinstance(elem, str) for elem in kwargs["testY"])
 
 
 @given(st.builds(ModelWithAlias))
 def test_build_using_different_signature_and_annotations(val):
     assert isinstance(val, ModelWithAlias)
+
+
+def use_bad_signature(self, testA: 1, testB: "not_a_type", *, testX: float):
```

Comment:
`flake8` complains here (`F821 undefined name 'not_a_type'`), because of the string-form of annotations used for forward references.  Another type, e.g. `("not a type", "nor a forward reference")`, should fix that.

Reply:
Huh, looks like I might be making a bug report upstream... Let's just use an integer as annotation then.

Reply:
Done :ok_hand: